examples/ingest/local/ingest.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# Processes the Unstructured-IO/unstructured repository
+# through Unstructured's library in 2 processes.
```

Comment:
Let's just have this example script process everything under `example-docs/` without the user having to edit it.
Please update this comment accordingly.